src/ralph/dns/templates/dns/dns_edit.html

```diff
@@ -0,0 +1,30 @@
+{% extends BASE_TEMPLATE %}
+{% load i18n %}
+
+{% block view_content %}
+  <div class="row">
+    <div class="large-12 columns">
+      <h1>DNS edit</h1>
+      <form method="post" action="">
+        {% csrf_token %}
+        {{ formset.management_form }}
+        <table>
+            <thead>
+              {% for name, field in formset.0.fields.items %}
+                <th>{{ field.label }}</th>
+              {% endfor %}
+            </thead>
+            {% for form in formset %}
+            <tr>
+              {% for field in form %}
+                <td>{{ field }} {{ field.errors }}</td>
+              {% endfor %}
+            </tr>
+            {% endfor %}
+        </table>
+        <br />
+        <input type="submit" class="button" value="{% trans "Save" %}" />
```

Comment:
is it possible to "wrap" it using standard ralph view to have buttons bar on the bottom?